src/button/button.scss

```diff
@@ -28,17 +29,48 @@ $iui-button-padding-large: $iui-xs * 6;
   white-space: nowrap;
   border: 1px solid transparent;
   @media (prefers-reduced-motion: no-preference) {
-    transition: color $iui-speed-fast ease-out,
-      background-color $iui-speed-fast ease-out,
+    transition: color $iui-speed-fast ease-out, background-color $iui-speed-fast ease-out,
       border-color $iui-speed-fast ease-out;
   }
 
-  > .iui-icon:not(.iui-user-icon) {
-    width: $iui-icons-default;
-    height: $iui-icons-default;
-    @media (prefers-reduced-motion: no-preference) {
-      transition: fill $iui-speed-fast ease-out;
+  &[disabled] {
+    @include iui-button-disabled;
+  }
+
+  @include themed {
+    background-color: t(iui-color-background-1);
+    border-color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-4));
+    color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-2));
+  }
```

Comment:
Dont think these are needed in this mixin.

Reply:
You're right.